Library/TelemetryClient.ts

```diff
@@ -217,6 +218,15 @@ class TelemetryClient {
             }
         }
 
+        // Sanitize tags and properties after running telemetry processors
+        if (accepted) {
+            envelope.tags = Util.validateStringMap(envelope.data.baseData.properties) as Tags & Tags[];
+
+            if (envelope.data.baseData.properties) {
```

Comment:
Because this code is executed after telemetry processors are executed we need to be extra careful with undefined references adding checks for envelope.data, and envelope.data.baseData